.github/workflows/docker-image-test.yml

```diff
@@ -33,8 +33,6 @@ jobs:
                   push: false
                   tags: posthog/posthog:testing
 
-            - name: Image digest
-              run: echo ${{ steps.docker_build.outputs.digest }}
```

Comment:
Removed this because apparently the digest is not being generated (perhaps because we don't push)